### PR TITLE
Add ownCloud motto change password email

### LINKED ISSUE #32042: Add ownCloud 'motto' to the footer of the ownCloud password changed successfully Confirmation email
**Mail of ownCloud Password Reset** ✅ 

screen shot 2018-07-13 at 11 50 12

**ownCloud password changed successfully** ❌ 

screen shot 2018-07-13 at 10 01 20

### PR DESCRIPTION
**Pull request by imujjwal96**
## Description

- Added the motto as footer
- Added alternative Plain text page for notify email
- Update notify email with html template.

## Related Issue

- Fixes #32042 

## How Has This Been Tested?

- test environment: http://35.154.227.4/core/

## Screenshots (if appropriate):
screen shot 2018-07-18 at 2 50 53 pm

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: core/templates/lostpassword/notify.php [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
+					<td bgcolor="<?php p($theme->getMailHeaderColor());?>">
+						<img src="<?php p(\OC::$server->getURLGenerator()->getAbsoluteURL(image_path('', 'logo-mail.gif'))); ?>" alt="<?php p($theme->getName()); ?>"/>
+					</td>
+				</tr>
+				<tr><td colspan="2">&nbsp;</td></tr>
+				<tr>
+					<td width="20px">&nbsp;</td>
+					<td style="font-weight:normal; font-size:0.8em; line-height:1.2em; font-family:verdana,'arial',sans;">
+						<?php
+						echo $l->t('Password changed successfully');
```
Comment (VicDeo) [inline core/templates/lostpassword/notify.php]: p($l->t('Password changed successfully'));

File: core/Controller/LostController.php [hunk 2]
```diff
@@ -262,12 +262,16 @@ protected function sendNotificationMail($userId) {
 		if ($email !== '') {
 			$tmpl = new \OC_Template('core', 'lostpassword/notify');
 			$msg = $tmpl->fetchPage();
+			$tmplAlt = new \OC_Template('core', 'lostpassword/altnotify');
+			$msgAlt = $tmplAlt->fetchPage();
+
```
Comment (VicDeo) [inline core/Controller/LostController.php]: two empty lines won't pass CI, please remove this one.

File: core/templates/lostpassword/altnotify.php [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<?php
+echo $l->t('Password changed successfully');
```
Comment (VicDeo) [inline core/templates/lostpassword/altnotify.php]: p($l->t('Password changed successfully'));

**Review by VicDeo — CHANGES_REQUESTED** [review]

**Comment by DeepDiver1975** [comment]
@imujjwal96 can I kindly ask you to adapt the unit tests? THX

```
1) Tests\Core\Controller\LostControllerTest::testSetPasswordSuccessful
Failed asserting that Array &0 (
    'status' => 'error'
    'msg' => 'Expectation failed for method name is equal to <string:setPlainBody> when invoked at sequence index 2
Parameter 0 for invocation OC\Mail\Message::setPlainBody('Password changed successfully...ud.org') does not match expected value.
Failed asserting that two strings are equal.'
) is identical to Array &0 (
    'status' => 'success'
).

/drone/src/tests/Core/Controller/LostControllerTest.php:536

**Comment by PVince81** [comment]
@imujjwal96 are you still interested in finishing this ? see comment above

**Pull request closed without merging**